bundles/binding/org.openhab.binding.tinkerforge/src/main/java/org/openhab/binding/tinkerforge/internal/config/ConfigurationHandler.java

```diff
@@ -335,7 +338,12 @@ private void createOHTFDeviceConfig(Map<String, String> deviceConfig) throws Con
                 || deviceType.equals(TypeKey.laser_range_finder_velocity.name())) {
             logger.debug("{} setting base config", LoggerConstants.CONFIG);
             TFBaseConfiguration tfBaseConfiguration = modelFactory.createTFBaseConfiguration();
-            if (deviceType.equals(TypeKey.bricklet_barometer)) {
+            if (deviceType.equals(TypeKey.backlight.name())) {
+                OHTFDevice<TFBaseConfiguration, LCDBacklightSubIds> ohtfDevice = modelFactory.createOHTFDevice();
+                ohtfDevice.getSubDeviceIds().addAll(Arrays.asList(LCDBacklightSubIds.values()));
+                ohtfDevice.setTfConfig(tfBaseConfiguration);
```

Comment:
Backlight is not MTFConfigConsumer therefore  no configuration can be set to OHTFDevice. I'm not sure if it will work if no config is set to OHTFDevice.